1657389 Add descriptions for SEO (series of related PRs) (#19571)
dotnet/docs: docs/csharp/programming-guide/classes-and-structs/object-and-collection-initializers.md

---
title: "Object and Collection Initializers - C# Programming Guide"

> description: Object initializers in C# assign values to accessible fields or properties of an object at creation without invoking a constructor and assignment statements.

Reviewer: In this context, object initializers are executed *after* the constructor:

```suggestion
description: Object initializers in C# assign values to accessible fields or properties of an object at creation after invoking a constructor.
```
